source/index.ts

```diff
@@ -211,11 +214,14 @@ export default class Audic extends EventTarget<{
 		void (async () => {
 			const vlc = await this._vlc;
 
-			const {repeat} = await vlc.info();
+			const {repeat, loop} = await vlc.info();
 
-			if (value !== repeat) {
+			if (repeat) {
```

Comment:
Shouldn't this also be like this?

```suggestion
			if (value !== repeat) {
```